### DISCUSSION TITLE
Ability to add multiple recipes to a single day in the Meal Planner

### DISCUSSION THREAD

**Discussion by BrandonG777** [mealie-recipes/mealie#442] [Ideas]
because people want sides with their entrée! 

and during weekends my kids expect me to feed them more than one meal per day... but at least I have a good out for the time being! :)

being able to randomly select certain tags would also be nice. Like being able to add 1 random dinner and 1 random side via the Meal Planner.

**Comment by hay-kot** [comment]
Check out the demo meal planner. I've added support for meals/sides and the ability to add random name/descriptions so you can just say "Veggies" as a side. 👍

**Reply by BrandonG777** [reply]
Nice implementation!

**Comment by Eyeball753** [comment]
Hi @hay-kot,

I was just about to make a similar suggestion until I saw this thread, might as well add to it. 

My idea is to have multiple categories in the meal planner section. You'd have 2 or 3 (I can't imagine needing more) rows and in each row you could select one of the categories you have selected in your options as meal planner categories. 
For example, the first row could randomize all breakfasts and the second row all main courses. 

Thank you for all the effort you've put into Mealie, I love it.
